crates/fuzzing/src/oracles/diff_wasmi.rs

```diff
@@ -13,18 +13,31 @@ pub struct WasmiEngine {
 impl WasmiEngine {
     pub(crate) fn new(config: &mut Config) -> Self {
         let config = &mut config.module_config.config;
-        config.reference_types_enabled = false;
         config.simd_enabled = false;
+        config.relaxed_simd_enabled = false;
         config.memory64_enabled = false;
-        config.bulk_memory_enabled = false;
         config.threads_enabled = false;
+        config.threads_enabled = false;
```

Comment:
This all looks right to me, thanks! The intention here is that the differential engine (wasmi here) both configures itself based on `Config` but additionally updates `Config` with anything it can't support. So for example simd is disabled here unconditionally in `Config` because wasmi doesn't support it, but you're doing the right thing otherwise where features are conditionally enabled in wasmi depending on `Config` for those that wasmi supports.

Two minor comments here:

* This is setting `threads_enabled = false` twice
* The `min_tables` and `max_tables` handling can probably be removed if wasmi supports reference types which is where multiple tables were added.

If you'd like, it might be good to leave some comments here too along the lines of:

```rust
// force generated modules to never have features that wasmi doesn't support
```

or something like that

Reply:
I have implemented your code review suggestions.